shed/index.go

```diff
@@ -158,6 +158,28 @@ func (f Index) Has(keyFields Item) (bool, error) {
 	return f.db.Has(key)
 }
 
+// HasMulti accepts multiple multiple key fields represented as Item to check if
```

Comment:
why do we need, on a par with put/get could just be subsumed under    `Has`?

Reply:
The main reason is that shed.Has is better for single chunk than shed.HasMulti, as it does not require a slice and does not have the iteration, making it more performant for single chunk checks.